<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDialog</class>
 <widget class="QWidget" name="AccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>485</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="6" column="1">
      <widget class="QLineEdit" name="edQuestion">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edProvider">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edPasswdLength">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbQuestion">
       <property name="text">
        <string>Sicherheitsfrage</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbAnswer">
       <property name="text">
        <string>Antwort</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edUsername">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbProvider">
       <property name="text">
        <string>Anbieter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbId">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edPassword">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="edAnswer">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edPasswdDefinition">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbPaswdLength">
       <property name="text">
        <string>Passwortl√§nge</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbPasswdDefinition">
       <property name="text">
        <string>Passwort Definition</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbPassword">
       <property name="text">
        <string>Passwort</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbUsername">
       <property name="text">
        <string>Benutzername</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lbLastModified">
       <property name="text">
        <string>Zuletzt modifiziert</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDateTimeEdit" name="edLastModified">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yy  -  HH:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopyPasswd">
       <property name="toolTip">
        <string>Das Password in die Zwischenablage kopieren</string>
       </property>
       <property name="text">
        <string>Passwort kopieren</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveData">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
